src/Claimer.sol

```diff
@@ -105,7 +92,7 @@ contract Claimer is Multicall {
         for (uint i = 0; i < _claimCount; i++) {
             fee += _computeFeeForNextClaim(minimumFee, decayConstant, perTimeUnit, elapsed, prizePool.claimCount() + i, maxFee);
         }
-        return fee;
+        return (fee / _claimCount) * _claimCount; // round it out to match claimPrizes()
```

Comment:
We lose a bit in precision here by dividing and then multiplying.
Why not just create a function `_computeFeePerClaim` that we could use in `claimPrizes`?

Reply:
It's supposed to lose precision!  That being said that's a good refactoring